wvd-templates/Create and provision WVD host pool/nestedtemplates/managedDisks-customimagevm.json

```diff
@@ -102,11 +103,21 @@
             "metadata": {
                 "description": "Location for all resources to be created in."
             }
+        },
+        "enablePersistentDesktop": {
+            "type": "bool",
+            "metadata": {
+                "description": "Set this parameter to true if you would like to enable Persistent Desktop experience. Defaults to false."
+            },
+            "defaultValue": false
         }
     },
     "variables": {
         "imageResourceId": "[resourceId(parameters('rdshImageSourceResourceGroup'), 'Microsoft.Compute/images', parameters('rdshImageSourceName'))]",
-        "storageAccountType": "[if( equals(parameters('rdshVMDiskType'), 'HDD'), 'Standard_LRS', 'Premium_LRS')]"
+        "storageAccountType": "[parameters('rdshVMDiskType')]",
+        "availabilitySetId": {
```

Comment:
Same as unamagedDisks-gallery.

Reply:
yep, that makes sense, I'll fix it.

Reply:
BTW, I found out why I did that way, there is no way to hide the id attribute without this extra logic and ARM does not accept an id=null value.